lib/xboxrt/libc_extensions/float.c

```diff
@@ -0,0 +1,166 @@
+#include <float.h>
+#include <stdlib.h>
+
+// Check architecture
+// Warning: This code also assumes a lack of SSE2+
+#if (!defined(__i386__))
+#error Unsupported architecture
+#endif
+
+#define _SW_INEXACT 0x00000001
+#define _SW_UNDERFLOW 0x00000002
+#define _SW_OVERFLOW 0x00000004
+#define _SW_ZERODIVIDE 0x00000008
+#define _SW_INVALID 0x00000010
+#define _SW_DENORMAL 0x00080000
```

Comment:
Wine uses these, but they don't seem to be from MS. But I was unable to find a MS FLOAT.H to verify.

We could also use `_EM_*`, but I felt it was ugly to assume that cw and sw used the same layout (even though they mostly do).

Reply:
CC @Thrimbor, master of Microsoft / mingw header files.